typescript/cli/src/config/multisig.ts

```diff
@@ -0,0 +1,117 @@
+import { confirm, input, select } from '@inquirer/prompts';
+import { z } from 'zod';
+
+import { ChainMap, ModuleType, MultisigIsmConfig } from '@hyperlane-xyz/sdk';
+import { objMap } from '@hyperlane-xyz/utils';
+
+import { errorRed, log, logBlue, logGreen } from '../../logger.js';
+import { runMultiChainSelectionStep } from '../utils/chains.js';
+import { FileFormat, mergeYamlOrJson, readYamlOrJson } from '../utils/files.js';
+
+import { readChainConfigIfExists } from './chain.js';
+
+const MultisigConfigMapSchema = z.object({}).catchall(
+  z.object({
+    type: z.string(),
```

Comment:
I think the complexity of multisig type shouldn't be surfaced to CLI users but we can punt on this

Reply:
Yeah fair. I do expect us to totally revisit everything ISM related very soon.